_health-care/_js/components/financial-assessment/AnnualIncomeSection.jsx

```diff
@@ -14,6 +16,26 @@ class AnnualIncomeSection extends React.Component {
   }
 
   // TODO: Figure out best way to enable users to change their response to pension
+
+  updateChildren() {
+    if (this.props.external === undefined || this.props.external.children === undefined) return;
+    if (this.props.external.children.length > 0) {
+      for (let i = 0; i < this.props.external.children.length; i++) {
+        const shortName = `${this.props.external.children[i].childFullName.first} ${this.props.external.children[i].childFullName.last}`;
+        if (this.props.data.children[i] === undefined) {
```

Comment:
I'm also confused why this is necessary. If `this.props.external.children.length` is greater than 0, when would there be a case where `this.props.data.children[i]` could be `undefined`?